### PR TITLE
Dropbox dependency removal

### PR DESCRIPTION
**Pull request by karyoutomoo**
Changed DropboxClient dependency to RestClient + Unit Testing for XDropBoxSdkImpl

### CODE REVIEW

File: lasvegas-ui-service/src/main/scala/com/gdn/lasvegas/ui/service/api/DropboxService.scala [hunk 1]
```diff
@@ -1,9 +1,15 @@
...
 import com.gdn.lasvegas.ui.model.order.DropboxInfoValueHistory
+import com.gdn.common.web.wrapper.response.GdnRestListResponse
+import com.gdn.lasvegas.ui.model.order.Order
+import com.gdn.lasvegas.ui.model.response.RestSingleResponse
 
 trait DropboxService {
   def getDropboxInfo(orderId: String): DropboxInfoValueHistory
   def resendPassword(bookingCode: String, customerEmail: String, provider: String): String
-
+  /*def getDropboxDetail(orderId: String): GdnRestSingleResponse[DropboxDetailResponse]
```
Comment (djokos77) [inline lasvegas-ui-service/src/main/scala/com/gdn/lasvegas/ui/service/api/DropboxService.scala]: delete unused code

File: lasvegas-ui-model/src/main/scala/com/gdn/lasvegas/ui/model/response/dropbox/ResendPasscodeResponse.scala [hunk 2]
```diff
@@ -0,0 +1,10 @@
+package com.gdn.lasvegas.ui.model.response.dropbox
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties
+//border javascala
+import com.gdn.common.web.base.BaseResponse
+//remove if not needed
+import scala.collection.JavaConversions._
```
Comment (djokos77) [inline lasvegas-ui-model/src/main/scala/com/gdn/lasvegas/ui/model/response/dropbox/ResendPasscodeResponse.scala]: dont use wildcard import

File: lasvegas-ui-model/src/main/scala/com/gdn/lasvegas/ui/model/response/dropbox/DropboxHistoryResponse.scala [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties
+
+import java.sql.Timestamp
+
+import com.gdn.common.web.base.BaseResponse;
+
+import scala.beans.{BeanProperty, BooleanBeanProperty}
+//remove if not needed
+import scala.collection.JavaConversions._
```
Comment (djokos77) [inline lasvegas-ui-model/src/main/scala/com/gdn/lasvegas/ui/model/response/dropbox/DropboxHistoryResponse.scala]: dont use wildcard import

File: lasvegas-ui-model/src/main/scala/com/gdn/lasvegas/ui/model/response/dropbox/DropboxHistoryResponse.scala [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+@SerialVersionUID(1L)
```
Comment (djokos77) [inline lasvegas-ui-model/src/main/scala/com/gdn/lasvegas/ui/model/response/dropbox/DropboxHistoryResponse.scala]: no need for UID

File: lasvegas-ui-model/src/main/scala/com/gdn/lasvegas/ui/model/response/dropbox/DropboxDetailResponse.scala [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+import java.sql.Timestamp
+
+import com.gdn.common.web.base.BaseResponse
+
+import scala.beans.{BeanProperty, BooleanBeanProperty}
+
+//remove if not needed
+import scala.collection.JavaConversions._
+
+@SerialVersionUID(1L)
```
Comment (djokos77) [inline lasvegas-ui-model/src/main/scala/com/gdn/lasvegas/ui/model/response/dropbox/DropboxDetailResponse.scala]: no need for UID

File: com/gdn/x/dropbox/service/rest/web/model/RunRetryResponse.java [hunk 6]
```diff
@@ -0,0 +1,33 @@
+package com.gdn.x.dropbox.service.rest.web.model;
```
Comment (djokos77) [inline com/gdn/x/dropbox/service/rest/web/model/RunRetryResponse.java]: convert to scala model

File: com/gdn/x/dropbox/service/rest/web/model/ResendPasscodeResponse.java [hunk 7]
```diff
@@ -0,0 +1,13 @@
+package com.gdn.x.dropbox.service.rest.web.model;
```
Comment (djokos77) [inline com/gdn/x/dropbox/service/rest/web/model/ResendPasscodeResponse.java]: convert to scala model

File: com/gdn/x/dropbox/service/rest/web/model/ResendPasscodeRequest.java [hunk 8]
```diff
@@ -0,0 +1,72 @@
+package com.gdn.x.dropbox.service.rest.web.model;
```
Comment (djokos77) [inline com/gdn/x/dropbox/service/rest/web/model/ResendPasscodeRequest.java]: convert to scala model

Comment (karyoutomoo) [reply com/gdn/x/dropbox/service/rest/web/model/ResendPasscodeRequest.java]: it is already located in com.gdn.lasvegas.ui.model.request

File: com/gdn/x/dropbox/service/rest/web/model/ResendPasscodeResponse.java (hunk 7 above)
Comment (karyoutomoo) [reply com/gdn/x/dropbox/service/rest/web/model/ResendPasscodeResponse.java]: it is already located in com.gdn.lasvegas.ui.model.response.dropbox

**Pull request merged**